tsdb_k8s_queries/challenges/default.json

```diff
@@ -326,4 +326,71 @@
         }{{ ", " if not loop.last else "" }}
         {%- endfor %}
       ]
-    }
+    },
+    {
+      "name": "storage",
```

Comment:
Cosmetic remark: conventional name here would be `append-no-conflicts-index-only`.